pkg/templates/nstemplatetiers/nstemplatetier_generator_whitebox_test.go

```diff
@@ -321,6 +390,63 @@ func TestNewNSTemplateTier(t *testing.T) {
 	})
 }
 
+func assertClusterResourcesTemplate(t *testing.T, decoder runtime.Decoder, actual templatev1.Template, tier string) {
+	expected := templatev1.Template{}
+	content, err := Asset(fmt.Sprintf("%s/cluster.yaml", tier))
+	_, _, err = decoder.Decode(content, nil, &expected)
+	require.NoError(t, err)
+	assert.Equal(t, expected, actual)
+	assert.Len(t, actual.Objects, 1)
+	cpuLimit := "1750m"
+	memoryLimit := "7Gi"
+	if tier == "team" {
+		cpuLimit = "2000m"
+		memoryLimit = "15Gi"
+	}
+	containsObj(t, actual, clusterResourceQuotaObj(cpuLimit, memoryLimit))
+}
+
+func assertNamespaceTemplate(t *testing.T, decoder runtime.Decoder, actual templatev1.Template, tier, kind string) {
+	content, err := Asset(fmt.Sprintf("%s/ns_%s.yaml", tier, kind))
+	require.NoError(t, err)
+	expected := templatev1.Template{}
+	_, _, err = decoder.Decode(content, nil, &expected)
+	require.NoError(t, err)
+	assert.Equal(t, expected, actual)
+	// Assert expected objects in the template
+	// Each template should have one Namespace, one RoleBinding, one LimitRange and three NetworkPolicy objects
+
+	// Namespace
+	containsObj(t, actual, namespaceObj(kind))
+	// RoleBinding "user-edit"
+	containsObj(t, actual, userEditRoleBindingObj(kind))
+
+	// LimitRange
+	cpuLimit := "150m"
+	memoryLimit := "512Mi"
+	if tier == "team" {
+		memoryLimit = "1Gi"
+	} else if kind == "dev" {
+		memoryLimit = "750Mi"
+	}
+	containsObj(t, actual, limitRangeObj(kind, cpuLimit, memoryLimit))
+
+	// NetworkPolicies
+	containsObj(t, actual, allowSameNamespacePolicyObj(kind))
+	containsObj(t, actual, allowFromOpenshiftIngressPolicyObj(kind))
+	containsObj(t, actual, allowFromOpenshiftMonitoringPolicyObj(kind))
+
+	// All templates in the "team" tier and "-code" templates in other tiers should also have additional RoleBinding and Role
+	if kind == "code" || tier == "team" || tier == "advanced" {
+		require.Len(t, actual.Objects, 8)
+		// Role & RoleBinding with additional permissions to edit roles/rolebindings
+		containsObj(t, actual, editRoleObj(kind))
+		containsObj(t, actual, userEditRoleBindingObj(kind))
```

Comment:
this should verify the RoleBinding for the `rbac-edit` role - I guess that you forgot to move it from the original test

Reply:
I don't understand your comment. Is it because the naming of the `editRoleObj ` func is wrong?

Reply:
btw, I renamed this func to `rbecEditRoleObj` in 8891e01

Reply:
no, I wanted to point out that you don't verify the `user-rbac-edit` RoleBinding. Instead of it you verify the `user-edit` RoleBinding. You even don't have a function that would create/return of that object - this is what I meant by "I guess that you forgot to move it from the original test"
https://github.com/codeready-toolchain/host-operator/blob/master/deploy/templates/nstemplatetiers/advanced/ns_code.yaml#L46-L57

Reply:
ok, I understand now. I've fixed that in f32789d

Reply:
Thanks, but you just added the `userRbacEditRoleBindingObj` function and you forgot to change it in the `containsObj` function:
```suggestion
		containsObj(t, actual, userRbacEditRoleBindingObj(kind))
```